<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadParametrizacoes</class>
 <widget class="QDialog" name="CadParametrizacoes">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametrização</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tensão</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtTensao"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Corrente</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtCorrente"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tempo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtTempo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>mGycm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtMgycm"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Tamanho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboTamanho">
       <item>
        <property name="text">
         <string>Pequeno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Médio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grande</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra Grande</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSalvar">
       <property name="text">
        <string>&amp;Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
